pkg/abstractions/image/credentials.go

```diff
@@ -23,8 +23,12 @@ func GetRegistryToken(opts *BuildOpts) (string, error) {
 		}
 	}
 
-	// Add additional registry checks here as needed.
-	return "", fmt.Errorf("unsupported registry or missing implementation")
+	// Default to Docker Hub
+	if token, err := GetDockerHubToken(opts); err == nil {
```

Comment:
Seems reasonable, would it be helpful to have a unit test here? 

Also, looks like python sdk tests are failing

Reply:
Yeah probably. Just added tests and fixed the python linting.

Reply:
👍